prerendering.bs

```diff
@@ -721,6 +748,56 @@ Add {{[DelayWhilePrerendering]}} to {{ScreenOrientation/unlock()}}.
 
 Add {{[DelayWhilePrerendering]}} to {{Navigator/requestMediaKeySystemAccess()}}.
 
+<h4 id="media-capture-patch">Media Capture and Streams</h4>
+
+Add {{[DelayWhilePrerendering]}} to {{Navigator/getUserMedia()}}, {{MediaDevices/getUserMedia()}} and {{MediaDevices/enumerateDevices()}}.
```

Comment:
I don't think this works for Navigator/getUserMedia(), since its API is callback-based rather than promise-based. (Well, it would work except that it would return a promise which it shouldn't.)

I'd suggest modifying the definition of `[DelayWhilePrerendering]` to only return the promise if the operation otherwise returns a promise, and otherwise to require that the operation returns `undefined`. (Or perhaps you have another alternative.)

Reply:
Yea, I think that would be a good way to go with `DelayWhilePrerendering`.

Reply:
@jeremyroman: changed `DelayWhilePrerendering` in new revision to also work for void functions.